source/components/organisms/PinInputModal/styled.ts

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components/native";
+import Body from "../../molecules/Dialog/Body";
+
+export const DialogContainer = styled(Body)`
+  text-align: center;
```

Comment:
Is text-align needed when you have text components that have `align="center"` ?

Reply:
If it's like in web then `text-align` is misleading since it actually aligns _elements_ per se, however in this case it is not actually needed since all elements seems to be full width. 

In fact, only `padding` does something here, as the rest is already inherited from `Body`. I'll clean it up!